### DISCUSSION TITLE
Vansh Chaurasiya Resume

### DISCUSSION THREAD

**Discussion by vansh-codes** [recodehive/Stackoverflow-Analysis#359] [Ideas]
Greetings Sir,
I would love to get a feedback on my resume.

Here's the link to my resume pdf: [VanshChaurasiya_2024.pdf](https://github.com/user-attachments/files/16387551/VanshChaurasiya_2024.pdf)

My linkedin: [Linkedin/vanshchaurasiya24](https://linkedin.com/in/vanshchaurasiya24)

My github: [Github/vansh-codes](https://github.com/vansh-codes/)

Thank you
Regards,
Vansh Chaurasiya

**Comment by sanjay-kv** [comment]
[ATS Version of CV (18).docx](https://github.com/user-attachments/files/16388165/ATS.Version.of.CV.18.docx)
[Uploading vansh.pdf…]()
